OrbitGrpcProtos/capture.proto

```diff
@@ -128,6 +128,48 @@ message GpuJob {
   uint64 dma_fence_signaled_time_ns = 11;
 }
 
+message GpuQueueSubmission {
+  GpuQueueSubmissionMetaInfo meta_info = 1;
+  repeated GpuSubmitInfo submit_infos = 2;
+  repeated GpuDebugMarker completed_markers = 3;
+  int32 num_begin_markers = 4;
+}
+
+message GpuQueueSubmissionMetaInfo {
+  int32 tid = 1;
+  uint64 pre_submission_cpu_timestamp = 2;
+  uint64 post_submission_cpu_timestamp = 3;
+}
+
+message GpuSubmitInfo {
+  repeated GpuCommandBuffer command_buffers = 1;
+}
+
+message GpuCommandBuffer {
+  uint64 begin_gpu_timestamp_ns = 1;
+  uint64 end_gpu_timestamp_ns = 2;
+}
+
+message GpuDebugMarkerBeginInfo {
+  GpuQueueSubmissionMetaInfo meta_info = 1;
+  uint64 gpu_timestamp_ns = 2;
+}
+
+message GpuDebugMarker {
+  GpuDebugMarkerBeginInfo begin_marker = 1;
+  uint64 end_gpu_timestamp_ns = 2;
+  uint64 text_key = 3;
+  int32 depth = 4;
+  Color color = 5;
+}
+
+message Color {
+  float red = 1;
```

Comment:
Clarify the range of the numbers in a comment. Probably [0.f, 1.f], but I've also seen [0.f, 255.f] being used (whether that is good practice is a different matter).

Reply:
Done.